vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/screenshot/AdamScreenCaptureTestRunListener.kt

```diff
@@ -56,7 +56,7 @@ class AdamScreenCaptureTestRunListener(
                 device.safePullFile(path, localFile.absolutePath)
                 logger.debug { "Received screen capture file $path" }
                 attachmentListeners.forEach {
-                    it.onAttachment(test.toTest(), Attachment(localFile, attachmentType))
+                    it.onAttachment(test.toTest(), Attachment(localFile, attachmentType, "screen"))
```

Comment:
`screen` should be a const or enum?

Reply:
every vendor can choose what to use as a name for the attachment. I doubt we should restrict anything here, but I can collect these values under an enum with optional overrides. For example log is named xcodebuild-log for apple and logcat for Android